src/write.c

```diff
@@ -1356,6 +1400,10 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
     if (hasGainMap) {
         AVIF_CHECKRES(avifValidateImageBasicProperties(firstCell->gainMap->image));
         AVIF_CHECKRES(avifValidateGrid(gridCols, gridRows, cellImages, /*validateGainMap=*/AVIF_TRUE, &encoder->diag));
+        if (firstCell->gainMap->image->colorPrimaries != 2 || firstCell->gainMap->image->transferCharacteristics != 2) {
```

Comment:
Any reason why this is `2` instead of `AVIF_TRANSFER_CHARACTERISTICS_UNSPECIFIED`?